### ISSUE TITLE
global: clean and document vagrant install process

### ISSUE THREAD

**Issue by nharraud** [EUDAT-B2SHARE/b2share#647]
There are currently two installation processes. One in install directory and the other in install-invenio2. The one documented, in invenio directory, is not working. We should remove it and document how to use install-invenio2.

**Comment by emanueldima** [comment]
Can you mention what installation folder have you used? Because we now have two install folders, and after switching to invenio2 the `install-invenio2` folder is the correct one.

**Comment by nharraud** [comment]
I used the install folder as described in the README. I will try with the install-invenio2.

**Comment by emanueldima** [comment]
The differences are quite small, but you are right (and the issue is valid) that there should be only one installation procedure.

**Comment by nharraud** [comment]
So if I understand correctly, the new workflow is just like this:

1. ON THE HOST: Clone git repository and goto the directory containing the Vagrantfile.
```
cd /path/to/your/development-repos/
git clone -b master git@github.com:EUDAT-B2SHARE/b2share.git
cd b2share/install-invenio2
```
2. ON THE HOST: Run `./install.sh`  in the folder created above, and then (after the provisioning is completed) login into the newly created machine.
```
vagrant ssh default
```
3. ON THE GUEST VM: Run start_b2share.sh, which will start the b2share server in development mode. You can now go on the host machine to http://localhost:4000 and the b2share/invenio site should show up.
```
# load virtualenv
workon b2share
# start service
/vagrant/start_b2share.sh
```

Am I missing something?
I also plan to remove the `install` directory and rename the `install-invenio2` one as `install`.

Would it break anything if we patched the 1.4 branch too?

**Comment by emanueldima** [comment]
That's correct. Unfortunately, this installation procedure is just for development, because in production Lassi (@llehtine) cannot rebuild the machine from scratch and uses our install scripts only as a guide. Therefore I don't think it would break anything to port this change to 1.4.
